activesupport/lib/active_support/core_ext/numeric/time.rb

```diff
@@ -1,6 +1,8 @@
 require 'active_support/duration'
 require 'active_support/core_ext/time/calculations'
 require 'active_support/core_ext/time/acts_like'
+require 'active_support/core_ext/date/calculations'
```

Comment:
Without requiring this, the example in [numeric.rb](https://github.com/tonytonyjan/rails/blob/445957690a861e953d2185f5e9f051350c12969d/activesupport/lib/active_support/core_ext/integer/time.rb#L13) couldn't even be executable.

It's If I only want the feature link "1.seconds.from_now", I have to only require one file:

```ruby
require 'active_support/core_ext/integer'
1.seconds.from_now
#=> 2015-01-19 20:09:37 +0800
```

But if I want something like `1.days.from_now`, I have to require 2 files:

```ruby
require 'active_support/core_ext/integer'
require 'active_support/core_ext/date/calculations'
1.days.from_now
#=> 2015-01-20 20:14:20 +0800
```

If we don't need add `calculations`, I wonder why we should add the example code that require additional dependency in comment. (maybe we should just remind people by adding some comment?)